x-pack/build_chromium/build.py

```diff
@@ -95,4 +109,4 @@ def archive_file(name):
 
 print('Creating ' + md5_filename)
 with open (md5_filename, 'w') as f:
-  f.write(md5_file(zip_filename))
+  f.write(md5_file(zip_filename))
```

Comment:
nit: no newline here, doesn't look like we lint python files, but wanted to point it out just in case.